### DISCUSSION TITLE
proxmox-backup-server

### DISCUSSION THREAD

**Discussion by lordyavin** [jiangcuo/Proxmox-Arm64#64] [Ideas]
Thank you very much for your efforts to build proxmox for arm! My goal was to use a Pi4 as backup-server but that packages are not ported yet. Do you plan to do this?

**Comment by TheBossME** [comment]
You're goin failed pbs is unofficial supported by JiangCuo and worked fine and tested with Raspi 4

**Comment by lordyavin** [comment]
@TheBossME, I'm sorry but I don't get it. [Proxmox does not provide ARM builds](http://download.proxmox.com/debian/pbs/dists/bookworm/pbs-no-subscription) and [jiangcuo ](https://global.mirrors.apqa.cn/proxmox/debian/pve/dists/bookworm/port/binary-arm64/) doesn't provide the proxmox-backup-server packages. So how did you install them without building them yourself?

**Reply by TheBossME** [reply]
I use this repo:

deb https://mirrors.apqa.cn/proxmox/debian/pbs bookworm port

Install Debian bookworm and add that repo worked on LXC and vm too 

No issues actually

**Reply by lordyavin** [reply]
didn't see that repo at all 🙈 Thanks
